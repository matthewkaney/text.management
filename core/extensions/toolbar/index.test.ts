/**
 * @jest-environment jsdom
 */

import { ToolbarMenu } from "./index";

describe("Toolbar Menu", () => {
  test("Snapshot test", () => {
    const menu = new ToolbarMenu("Test", []);
    expect(menu.dom).toMatchSnapshot();
  });
});
