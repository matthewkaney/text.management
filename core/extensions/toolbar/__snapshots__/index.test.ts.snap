// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar Menu Snapshot test 1`] = `
<div
  class="cm-menu"
  role="none"
>
  <button
    class="cm-menu-trigger"
    id="Test"
    role="menuitem"
    tabindex="0"
  />
  <div
    aria-labelledby="Test"
    class="cm-menu-item-list"
    role="menu"
  >
    <div
      class="cm-menu-item-group"
    />
  </div>
</div>
`;
